h2o-bindings/bin/custom/python/gen_infogram.py

```diff
@@ -42,6 +42,7 @@ def plot(self, train=True, valid=False, xval=False, figsize=(10, 10), title="Inf
         
         plt = get_matplotlib_pyplot(server, raise_if_not_available=True)
         polycoll = get_polycollection(server, raise_if_not_available=True)
+        np = _get_numpy("Infogram plots")
```

Comment:
Maybe we should rename `_get_numpy` to `get_numpy` to match the other `get_{library}` functions.

Reply:
Sure, though the function is slightly different because it has different arguments.  Should we standardize it to match the others?  

`_get_numpy` asks the user for a specific function to reference (as a required arg) which does not make a lot of sense since when you import the library, you probably want to use a variety of functions.  It's also weird because even though we have the input text string, in every case where it's used, we are unnecessarily hardcoding a different print message (so you'd get duplicate messages anwyay).  Exmaple: https://github.com/h2oai/h2o-3/blob/ef6070c3d5d9a73c674b908db2b90e8496db14cc/h2o-py/h2o/model/model_base.py#L1468-L1471

I am also ok to just merge this now so it can get into 3.36.0.2, and then rename later.